python/tskit/trees.py

```diff
@@ -1205,6 +1205,27 @@ def right_sib_array(self):
         """
         return self._right_sib_array
 
+    def siblings(self, u):
+        """
+        Returns the siblings of the specified node ``u`` as a tuple of integer
+        node IDs. If ``u`` has no siblings or is not a node in the current tree,
+        return an empty tuple. If the tree has multiple roots, then the siblings
+        of a root is the other roots, which are related by a virtual root.
+        Also, the virtual root itself has no siblings, and so an empty tuple is
+        returned. The ordering of siblings is arbitrary and should not be depended on;
+        see the :ref:`data model <sec_data_model_tree_structure>` section for details.
+
+        :param int u: The node of interest.
+        :return: The siblings of ``u``.
+        :rtype: tuple(int)
+        """
+        parent = self.parent(u)
+        if self.has_multiple_roots and u in self.roots:
```

Comment:
This could be inefficient as the list of roots can be large (in a "missing" tree eg.)

How about:
```python
if u == self.virtual_root:
    return tuple()
parent = self.parent(u)
if parent == tskit.NULL:
     parent = self.virtual_root
return tuple(v for v in self.children(parent) if u != v)
```